src/common/BrowserStorage.ts

```diff
@@ -37,6 +38,11 @@ export type CorrectItem = {
 	url: string;
 };
 
+export type ScrobblingItem = Omit<ItemBase, ''> & {
```

Comment:
I don't get this Omit-pattern. As nothing is omitted, isn't ScrobblingItem the same as ItemBase except for the extension at the end?

Reply:
If you don't do it like this, `browser.storage.local.set` doesn't like it, for some reason. I think it's because it's an interface, and the empty Omit changes it to an actual object?